src/benchmarklib/abstract_table_generator.hpp

```diff
@@ -43,14 +53,9 @@ class AbstractTableGenerator {
    */
   virtual std::unordered_map<std::string, BenchmarkTableInfo> generate() = 0;
 
- protected:
-  /**
-   * The AbstractTableGenerator stores the whole BenchmarkConfig, yet not all fields in BenchmarkConfig influence
-   * table generation.
-   * This helper function creates a BenchmarkConfig with the fields that affect table generation
-   */
-  static std::shared_ptr<BenchmarkConfig> _create_minimal_benchmark_config(uint32_t chunk_size);
```

Comment:
what happened to this?

Reply:
Apparently noone was calling it, it wasn't implemented, so why keep it.